samples/snippets/csharp/VS_Snippets_VBCSharp/vblinkcompilercs/cs/utility.cs

```diff
@@ -8,17 +7,14 @@
 public class Utility
 {
     // The following code causes an error when called by a client assembly.
-    public List<Range> GetRange1() {
-        //</Snippet2>
+    public List<Range> GetRange1()
+	{
         return null;
-        //<Snippet3>
     }
 
     // The following code is valid for calls from a client assembly.
-    public IList<Range> GetRange2() {
-        //</Snippet3>
+    public IList<Range> GetRange2()
+	{
```

Comment:
```suggestion
    public IList<Range> GetRange2()
    {
```

Do not use tabs, instead use 4x spaces.